frontend/javascripts/admin/api_flow_types.js

```diff
@@ -578,6 +586,7 @@ export type ServerSkeletonTracingTree = {
 
 export type ServerTracingBase = {|
   id: string,
+  userBoundingBoxes: Array<ServerUserBoundingBox>,
   userBoundingBox?: ServerBoundingBox,
```

Comment:
I suggest to keep the property. The `userBoundingBox` property can always be part of tracing send by the server to stay backward compatible. 
Additionally the type `ServerTracingBase` is only used when parsing protobuf tracings and won't be used later on. Thus it should be fine and correct to keep the property in this type.

Nevertheless, I tried getting rid of it. This made the code harder to read because I had to cast between types and stuff like that. :confused: